server/package.json

```diff
@@ -28,7 +28,7 @@
     "typeorm:migrations:generate": "typeorm migration:generate -d ./dist/infra/database.config.js",
     "typeorm:migrations:run": "typeorm migration:run -d ./dist/infra/database.config.js",
     "typeorm:migrations:revert": "typeorm migration:revert -d ./dist/infra/database.config.js",
-    "typeorm:schema:drop": "typeorm schema:drop -d ./dist/infra/database.config.js",
+    "typeorm:schema:drop": "typeorm query -d ./dist/infra/database.config.js 'DROP schema public cascade; CREATE schema public;'",
```

Comment:
Depending how long this PR takes this would also be good to just fix on main immediately.

Reply:
Feel free to just cherry pick this onto main

Reply:
Cool, sounds good.